go.mod

```diff
@@ -1,143 +1,63 @@
 module github.com/cloudtrust/keycloak-bridge
 
-go 1.15
+go 1.17
+
+require (
+	github.com/cloudtrust/common-service/v2 v2.5.5
+	github.com/cloudtrust/httpclient v1.0.7
+	github.com/cloudtrust/keycloak-client/v2 v2.5.3
+	github.com/go-kit/kit v0.12.0
+	github.com/go-sql-driver/mysql v1.6.0
+	github.com/golang/mock v1.6.0
+	github.com/google/uuid v1.3.0
+	github.com/gorilla/mux v1.8.0
+	github.com/lib/pq v1.10.4
+	github.com/pkg/errors v0.9.1
+	github.com/rs/cors v1.8.2
+	github.com/spf13/cast v1.4.1
+	github.com/spf13/pflag v1.0.5
+	github.com/spf13/viper v1.10.1
+	github.com/stretchr/testify v1.7.0
+	golang.org/x/time v0.0.0-20211116232009-f0f3c7e86c11
+	gopkg.in/h2non/gentleman.v2 v2.0.5
+)
 
 require (
```

Comment:
There are 2 "require" blocs in this file?

Reply:
Yes in every go.mod files now with go1.17
One require for direct dependencies and one for indirect ones